docs/src/examples/dimetourrotations.md

```diff
@@ -0,0 +1,161 @@
+
+Now, let's place this algebra element on the manifold using the exponential map [`exp`](@ref):
+```julia
+xRi = exp(G, xR0, xXi)
+# similarly for known underlying manifold
+xRi_ = exp(M, xR0, xXi)
+
+@assert isapprox( xRi, xRi_ )
+```
+
+### To Coordinates
+
+The logarithm transform from the group back to algebra (or coordinates) is:
+```julia
+xXi_ = log(G, xR0, xRi)
+xXi__ = log(M, xR0, xRi)
+@assert xXi == xXi__
+```
+
+Similarly, the coordinate value can be extracted from the algebra using [`vee`](@ref), or directly from the group using the more generalized [`get_coordinates`](@ref):
+```julia
+# extracting coordinates using vee
+xθi__ = vee(G, xR0, xXi_)[1]
+_xθi__ = vee(M, xR0, xXi_)[1]
+
+# OR, the preferred generalized get_coordinate function
+xθi_ = get_coordinates(G, xR0, xXi_, B)[1]
+_xθi_ = get_coordinates(M, xR0, xXi_, B)[1]
+
+# confirm all versions are correct
+@assert isapprox( xθi, xθi_ ); @assert isapprox( xθi, _xθi_ )
+@assert isapprox( xθi, xθi__ ); @assert isapprox( xθi, _xθi__ )
+```  
+
+!!! note
+    The disadvantage might be that the representation of `X` is not nice, i.e. it uses too much space or doing vector-calculations is not so easy.  E.g. fixed rank matrices are overloaded for all vector operations, but maybe that is “not enough” for a general user application that really wants vectors. But: Given a basis `B` one can look at the coefficients of the tangent vector `X` with respect to basis `B`.  From the Sphere example note above the basis would be `Y1=[0.0,1.0,0.0]` and `Y2=[0.0,0.0,1.0]`, the so to get the coordinates would be `c = get_coordinates(Sphere(2), p, X, B)`.  Visa versa, if you have a coordinate vector with respect to a basis `B` of the tangent space at `p` and want the vector back, then you do `X2 = get_vector(M, p, c, B)` (and you should have `X2==X`).  The coordinate vector `c` might also have the advantage of saving memory. E.g. SPD matrix tangent vectors take n^2 entries to save, i.e. storing the full matrix, but the coordinate vectors only take n(n+1)/2.
+
+### Actions and Operations
+
+With the basics in hand on how to move between the coordinate, algebra, and group representations, let's briefly look at composition and application of points on the manifold.  For example, a `Rotation(n)` manifold is the mathematical representation, but the points have an application purpose in retaining information regarding a specific rotation.     
+
+Points from a Lie group may have an associated action (i.e. a rotation) which we [`apply`](@ref).  Consider rotating through `θ = π/6` three vectors `V` from their native domain `Euclidean(2)`, from the reference point `a` to a new point `b`.  Engineering disciplines sometimes refer to the action of a manifold point `a` or `b` as reference frames.  More generally, by taking the tangent space at point `p`, we are defining a local coordinate frame with basis `B`, and should not be confused with "reference frame" `a` or `b`.
```

Comment:
In the language of Manifolds.jl, what the code below does is rotating three points `aV1`, `aV2` and `aV3` from `Euclidean(2)` around the origin (point `[0, 0]`). These points can, of course, be identified with vectors (specifically, [free vectors](https://en.wikipedia.org/wiki/Euclidean_vector#Overview) that have two coordinates. You can also keep separately their origins if they are meant to represent [bound vectors](https://en.wikipedia.org/wiki/Euclidean_vector#In_Cartesian_space). FYI, I'm not particularly attached to this bound/free vector terminology so feel free to use different names, I just added these links to make the point more clear.

Manifolds.jl also offers a more differential-geometric way for representing bound vectors. Vectors `aV1`, `aV2` and `aV3` that are bound at `a` can be thought of as belonging to the tangent space at `a`, that is `TangentSpaceAtPoint(Euclidean(2), a)`. If you then wanted to rotate `aV1`, `aV2` and `aV3` through a rotation that sends `a` to `b`, you'd actually apply _differential_ of the group action on them, more or less like that:
```julia
using Manifolds, StaticArrays

a = [2, 3]
aV1 = [1;0]
aV2 = [0;1]
aV3 = [10;10]
G = SpecialOrthogonal(2)
A_left = RotationAction(Euclidean(2), G)
bθa = π/6

p0 = @SMatrix [1.0 0; 0 1]
xR0 = identity(G, p0)

B = DefaultOrthogonalBasis()
bXa = get_vector(base_manifold(G), xR0, bθa, B)
bRa = exp(G, xR0, bXa)

b = apply(A_left, bRa, a)
bV1 = apply_diff(A_left, bRa, a, aV1)
```
Of course, this is an extremely convoluted way of rotating a vector but that's what actually can be extended to arbitrary manifolds and actions. I don't know how useful would that perspective be, so don't take it as a recommendation for this tutorial, just a point that might be worth keeping in mind.

Reply:
> offers a more differential-geometric way for representing bound vectors

I agree with that, let me try work this in.  Will take a bit before I can though, busy week coming up this side.

> I don't know how useful would that perspective be, so don't take it as a recommendation for this tutorial, just a point that might be worth keeping in mind.

I think it is important to get the language right, otherwise this will always be something that comes up with more people wanting to add contributions.  Better to get it right now :-)  thanks!